Convert dagobah version of the Data Managers tutorial (#3778)
galaxyproject/training-material: topics/admin/tutorials/cvmfs/tutorial.md

> well. So we're just going to get on with things.
{: .spoken data-visual="gtn" data-target="#cvmfs-quote"}

> A slideshow presentation on this subject can be found [here](slides.html). More details on the usegalaxy.org (Galaxy Main's) reference data setup and CVMFS system can be found [here](https://galaxyproject.org/admin/reference-data-repo/#usegalaxyorg-reference-data).

Reviewer: ```suggestion
[A slideshow presentation on this subject]({% link topics/admin/tutorials/cvmfs/slides.html %}) is available. More [details are available on usegalaxy.org (Galaxy Main's) reference data setup](https://galaxyproject.org/admin/reference-data-repo/#usegalaxyorg-reference-data) and CVMFS system.
```
